src/Lykke.Job.TransactionHandler/Modules/JobModule.cs

```diff
@@ -181,7 +181,12 @@ private void BindRepositories(ContainerBuilder builder)
                 new MarketOrdersRepository(AzureTableStorage<MarketOrderEntity>.Create(_dbSettingsManager.ConnectionString(x => x.HMarketOrdersConnString), "MarketOrders", _log)));
 
             builder.RegisterInstance<ILimitOrdersRepository>(
-                new LimitOrdersRepository(AzureTableStorage<LimitOrderEntity>.Create(_dbSettingsManager.ConnectionString(x => x.HMarketOrdersConnString), "LimitOrders", _log)));
+                new LimitOrdersRepository(
+                    AzureTableStorage<LimitOrderEntity>.Create(
+                        _dbSettingsManager.ConnectionString(x => x.HMarketOrdersConnString),
+                        "LimitOrders",
+                        _log,
+                        TimeSpan.FromSeconds(30))));
```

Comment:
This is not the way how we should fix performance problems.  Also, this is not a 'test' branch, not 'hotfix'.